numpy/core/src/umath/loops.c.src

```diff
@@ -20,6 +20,12 @@
 
 #include <string.h> /* for memchr */
 
+/* Use Libdivide for faster division */
+/* TODO Explore placing specialised versions in `numpy/core/src/common/simd` */
```

Comment:
```suggestion
```
Lets remove the TODO, I think adding SIMD is a common TODO and probably needs no specific nudge.